src/command_modules/azure-cli-network/azure/cli/command_modules/network/tests/latest/test_network_commands.py

```diff
@@ -1677,6 +1677,63 @@ def test_network_vnet_peering(self, resource_group):
         self.cmd('network vnet peering delete -g {rg} -n peering2 --vnet-name vnet2')
         self.cmd('network vnet-gateway delete -g {rg} -n gateway1')
 
+    @ResourceGroupPreparer(name_prefix='cli-test_vnet_peering_group_2', parameter_name='resource_group_2')
+    @ResourceGroupPreparer(name_prefix='cli-test_vnet_peering_group_1', parameter_name='resource_group_1')
+    def test_network_vnet_peering_different_resource_groups(self, resource_group_1, resource_group_2):
```

Comment:
Fine to test the cross-RG scenario, but probably unnecessary. More important to test that name or ID work for the parameter.